### PR TITLE
Zoowsome part 2 (without twist) finished

### PR DESCRIPTION
**Pull request by ASabau**
I hava a problem in the project: I have created in main 50 animals and 10 caretakers but when I run the Russian-roulette with animals and caretakers code, all my caretakers die by the end of that code(of course, at run-time).... The problem, I think, is at the kill() method from the Animal class. If I return false everytime from that method everything is ok.

Could you please take a look over the code, if you have time, and tell me if it is ok?
Thanks in advance!

### CODE REVIEW

**Comment by AlexBondor** [comment]
Maybe the problem is that your 50 animals require more attention than 10 caretakers can offer with their working hours.

**Comment by AlexBondor** [comment]
The `kill()` method is alright.. no problems there. Try with 10 animals and 20 caretakers.. if all your caretakers are dying as well then there might be a problem in the code. Then try creating only non dangerous animals and test again.

**Comment by AlexBondor** [comment]
Also.. you should've worked on the previous part of this project.. move the code you have changed now in the old folder.. work on that specific folder from now on for the new assignments on the Zoo project

**Comment by ASabau** [comment]
I have tried with 20 caretakers and 10 animals and at least 8 caretakers die each time... Also, I have tried with non-dangerous animals, and yes, no caretaker dies this time. So, if at least 8 caretakers die each time if I have dangerous animals (Tiger, Shark, etc) is it a problem with the code?

And regarding the folder, can I replace the Zoowsome folder that I have now, with the updated one? Because I don't have the workspace in my repository, so the files don't get updated, and I have created packages and classes and it would be easier to delete the old folder and replace it with the updated one. Should I do that?

Thank you for your feedback!

**Comment by AlexBondor** [comment]
Well.. you can do something else.. From eclipse go to File > Import.. and import the old project from your own repository.. make the changes there then remove the new folder you created. This way I will only see the new changes when you do a new pull request.. otherwise.. everytime you change the whole folder with the new code I will get all the files and it will be really hard to track the changes you have made. The code is not wrong.. It is just that we are using random numbers..

File: Students/Sabau M. Andrei/Zoowsome/src/javasmmr/zoowsome/models/employees/Employee.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		int nrOfDigits = 0;
+		long copyOfNr = nr;
+		
+		while(copyOfNr > 0){
+			nrOfDigits++;
+			copyOfNr /= 10;
+		}
+		
+		if(nrOfDigits == 13)
+			this.id = nr;
```
Comment (AlexBondor) [inline Students/Sabau M. Andrei/Zoowsome/src/javasmmr/zoowsome/models/employees/Employee.java]: Use braces..

File: Students/Sabau M. Andrei/Zoowsome/src/javasmmr/zoowsome/models/employees/Employee.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	public void setName(String name){
+		this.name = name;
+	}
+	
+	public long getId(){
+		return id;
+	}
+	
+	public void setId(){
+		long nr = (long) (((double) 1e14 )* Math.random()) ;
```
Comment (AlexBondor) [inline Students/Sabau M. Andrei/Zoowsome/src/javasmmr/zoowsome/models/employees/Employee.java]: Better take this logic out in some other method like `generateId()` and then pass the result as argument to your `setId()` method

**Pull request merged**